mobile/lib/main.dart

```diff
@@ -503,19 +508,44 @@ Future<void> runBackend(bridge.Config config) async {
 
   FLog.info(text: "App data will be stored in: $appDir");
   FLog.info(text: "Seed data will be stored in: $seedDir");
-
-  await initFirebase();
-  await requestNotificationPermission();
-
   await startBackend(config: config, appDir: appDir, seedDir: seedDir);
 }
 
+FlutterLocalNotificationsPlugin initLocalNotifications() {
+  final flutterLocalNotificationsPlugin = FlutterLocalNotificationsPlugin();
+  const androidSettings = AndroidInitializationSettings('@mipmap/ic_launcher');
+  const darwinSettings = DarwinInitializationSettings();
+  const initializationSettings =
+      InitializationSettings(android: androidSettings, macOS: darwinSettings, iOS: darwinSettings);
+  flutterLocalNotificationsPlugin.initialize(initializationSettings);
+  return flutterLocalNotificationsPlugin;
+}
+
 Future<void> initFirebase() async {
   await Firebase.initializeApp(
     options: DefaultFirebaseOptions.currentPlatform,
   );
 }
 
+Future<void> configureFirebase(FlutterLocalNotificationsPlugin localNotifications) async {
+  // Configure message handler
+  FirebaseMessaging.onMessage.listen((RemoteMessage message) {
+    // TODO: Handle messages from Firebase
+    FLog.debug(text: "Firebase message received: ${message.data}");
+
+    FLog.debug(text: "Message data: ${message.data}");
```

Comment:
Nit: why are you printing the same content twice?

Reply:
oops! removed